plugin.video.sgtv/default.py

```diff
@@ -108,18 +107,31 @@ def channelShows(channel, page=0):
 	if current_page < max_page:
 		page = str(current_page + 1)
 		addXBMCItem (__language__(31000), "", "?mode=loadChannel&channel="+channel+"&page="+page, True)
-
 	xbmc.executebuiltin("Container.SetViewMode(500)")
```

Comment:
Please drop any calls to SetViewMode from the code. Forcing skin views are not allowed on the repository anymore I'm afraid. Skin view id's also depend on the skin the user is running. So please use xbmcplugin.setContent to define the correctly content type (tvshows for instance). Also please set the mediatype infolabel when building the ListItem object